app/backend/src/app/auth/auth.service.ts

```diff
@@ -81,5 +95,63 @@ export class AuthService {
     };
   }
 
-  
+  async signInWithEmail(signInDto: SignInDto): Promise<{ student: Partial<Student>; token: string }> {
+    const { email, password } = signInDto;
+    const student = await this.studentRepository.findOne({ where: { email } });
```

Comment:
we are not finding the user on the base of email. We need to find the user on googleId